content/05.proteolysis.md

```diff
@@ -8,13 +8,65 @@ Peptides that are too short are less likely to uniquely match to a single protei
+But the drawback of trypsin is that majority (56%) of the tryptic peptides are ≤ 6 amino acids, and hence using trypsin alone limits the observable proteome [@PMID: 20113005; @PMID: 25823410; @PMID:30687733].
+This limits the number of identifiable protein isoforms and post translational modifications.
+
 
 3. theoretical studies of proteolysis enzymes [@DOI:https://doi.org/10.1155/2014/960902]
 4. Challenges associated with alternative enzyme choices (non-specific and semi-specific enzymes)
-5. Alternative enzyme choices (one paragraph each?) - LysC
-6. GluC
-7. AspN
-8. Alpha-lytic protease [@DOI:https://doi.org/10.1074/mcp.m113.034710] and how it enables mapping human SUMO sites [@PMID:29079793].
-9. others?
+5. Alternative enzyme choices (one paragraph each?) - 
+The enzyme choice mostly depends on the application.
+In general, for a mere protein identification mostly trypsin is the choice due to the reasons aforementioned.
+However, alternative enzymes are used for de novo assembly when the genomic data information is limited in the public database repositories.
+Especially, they serve as an advantage in improving the sequence coverage [@PMID:12643544; @PMID:20113005] and also helps in distinguishing protein isoforms [@PMID:27123950].
+In addition to that they also helps in improving the sensitivity and accuracy of protein quantification in terms of the number of petides derived [@PMID:30336047].
+6. Lys-C:
+Lysyl endopeptidase (Lys-C) obtained from  _Lysobacter enzymogenesis_ is a serine protease invloved in cleaving carboxyl terminus of lysine [@PMID:25823410].
+Like trypsin, the optimum pH required for its activity is from pH 7-9. 
+One among the major advantage of Lys-C is that it is resistant to denaturing agents, including 8 M urea [@PMID: 27123950].
+Compared to Arg residues, trypsin is less efficient in cleaving Lys.
+This also interfere in protein quantification in terms of sensitivity and accuracy.
+Hence to achieve complete protein digestion with minimal missed cleavages, Lys-C is often used in parallel to complement tryspin digestion [@PMID:23017020; @PMID:16512685]. 
+7. Glu-C:
+Glutamyl peptidase I, commonly known as Glu-C or V8 protease, is a serine protease obtained from _Staphyloccous aureus_.
+This specifically cleaves at the C-terimus of Glu. 
+However, depending on the pH conditions, in addition to Glu (pH 4.0) it is invloved in cleaving Asp (pH 8.0) residues also [@PMID:4627743].
+8. Asp-N:
+Peptidyl-Asp metallopeptidase, commonly known as Asp-N, is a metalloprotease obtained from _Pseudomonas fragi_ [@PMID:2669754].
+It is involved in the hydrolysis of Asp residues at the N-terminal of the peptide and works at an optimum pH 4-9.  
+Studies also suggests that Asp-N cleaves at the amino terminus of Glu when a detergent is present in the proteolysis buffer [@PMID:2669754]. 
```

Comment:
The authors suspected that the N-terminal glu residue might have resulted due the presence of SDS in the buffer.
In Page 1533, the very first line of the article says like this...." **Alternatively, the use of sodium dodecyl sulfate in our study may have altered the specificity, although no effect of this detergent on Endoproteinase Asp-N activity at
the concentrations used here (0.01%) has been detected"**

 The authors also stated that this might be also arising due to the difference in pH and other buffer conditions.